lib/Schema.js

```diff
@@ -181,9 +181,11 @@ Schema.prototype.toDynamo = async function(model, options) {
 
 Schema.prototype.parseDynamo = async function(model, dynamoObj) {
 
+  debug('MADE IT')
   for(const name in dynamoObj) {
     let attr = this.attributes[name];
 
+    if (!attr && this.options.errorUnknown) throw new errors.ParseError(`unknown top-level attribute ${name} on model ${model.$__.name}: ${JSON.stringify(dynamoObj[name])}`)
```

Comment:
Dynamoose parses top-level attributes and nested attributes in different ways. Top level attributes are handled via `Schema#parseDynamo` while nested fields are handled via `Attribute#parseDynamo`. The above code handles the case of encountering an unknown "top-level" attribute in a DynamoDB record.

Name is the field name that is unknown.

model.$__.name is the model name (in case you are doing operations with multiple models, it is useful to know which one has a problem).

`JSON.stringify(dynamoObj[name])` will simply resolve to the value of the unknown field. I should probably add `with value: ` to the error string. The reason to use `JSON.stringify` is because if the value is an object then all that will be logged is `object [object]`.